play-services-location/core/provider/src/main/kotlin/org/microg/gms/location/network/cell/CellDetailsSource.kt

```diff
@@ -11,8 +11,11 @@ import android.os.Build.VERSION.SDK_INT
 import android.os.WorkSource
 import android.telephony.CellInfo
 import android.telephony.TelephonyManager
+import android.util.Log
 import androidx.core.content.getSystemService
 
+const val TAG = "CellDetailsSource"
```

Comment:
If the log tag only applies to the current file, it should be private.

Reply:
Done